packages/image-loader/index.js

```diff
@@ -13,7 +13,7 @@ module.exports = ({ config }, options = {}) => {
 
   config.module
     .rule('img')
-    .test(/\.(png|jpg|jpeg|gif)(\?v=\d+\.\d+\.\d+)?$/)
+    .test(/\.(png|jpg|jpeg|gif|webp)(\?v=\d+\.\d+\.\d+)?$/)
```

Comment:
Looks like the master commit got re-included here. You may want to:

```sh
git checkout master
git pull upstream master
git checkout style-loader-extract
git checkout HEAD~1
git rebase master
git commit -m "Pass options.hot instead of options.cssHotLoader" --amend
git push -f origin style-loader-extract
```

Reply:
Whups - yeah I rebased from master